packages/terra-notification-dialog/src/clinical-lowlight-theme/NotificationDialog.module.scss

```diff
@@ -0,0 +1,48 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-notification-dialog-background: #222a2e;
+    --terra-notification-dialog-border-radius: 3px;
+    --terra-notification-dialog-small-horizontal-inset: 17px;
+    --terra-notification-dialog-max-width: 500px;
+    --terra-notification-dialog-floating-background-height: 5.78571rem;
+    --terra-notification-dialog-alert-background-image: linear-gradient(to bottom, #272d31, #222a2e);
+    --terra-notification-dialog-alert-border-top: 7px solid #fb4c4c;
+    --terra-notification-dialog-error-background-image: linear-gradient(to bottom, #1f2629, #222a2e);
+    --terra-notification-dialog-error-border-top: 7px solid #141719;
+    --terra-notification-dialog-warning-background-image: linear-gradient(to bottom, #383733, #222a2e);
+    --terra-notification-dialog-warning-border-top: 7px solid #ffb166;
+    --terra-notification-dialog-info-background-image: linear-gradient(to bottom, #1f2629, #222a2e);
+    --terra-notification-dialog-info-border-top: 7px solid #141719;
+    --terra-notification-dialog-custom-background-image: linear-gradient(to bottom, #1f2629, #222a2e);
+    --terra-notification-dialog-custom-border-top: 7px solid #141719;
+    --terra-notification-dialog-header-padding: 0 1.42857rem;
+    --terra-notification-dialog-header-content-border-bottom: 1px solid #141719;
+    --terra-notification-dialog-header-content-padding: 1.21428rem 0 0.5rem;
+    --terra-notification-dialog-header-container-margin-left: 0.5rem;
+    --terra-notification-dialog-signal-word-color: #b2b5b6;
+    --terra-notification-dialog-signal-word-font-size: 1.42857rem;
+    --terra-notification-dialog-signal-word-min-height: 1.64285rem;
+    --terra-notification-dialog-title-color: #b2b5b6;
+    --terra-notification-dialog-title-font-size: 1.2857rem;
+    --terra-notification-dialog-title-font-weight: normal;
+    --terra-notification-dialog-title-margin: 0;
+    --terra-notification-dialog-title-min-height: 1.5rem;
```

Comment:
Needs `terra-notification-dialog-title-margin-top` to satisfy #1175 .

Reply:
I double-checked my changes, `terra-notification-dialog-title-margin-top` styles have removed: https://github.com/cerner/terra-framework/pull/1105/files#diff-199a5a3214c6de6b926614643935d3a5L102